packages/expo-flutter-adapter/example/ios/Flutter/flutter_export_environment.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+# This is a generated file; do not edit or check into version control.
```

Comment:
> or check into version control

🙂

Reply:
Ah haha this wasn't in the gitignore when I first generated the plugin so this must be something new Flutter added. Will remove and add to the gitignore now!